packages/testing/package.json

```diff
@@ -6,7 +6,7 @@
     "dist"
   ],
   "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "types": "src/index.d.ts",
```

Comment:
```suggestion
  "types": "dist/index.d.ts",
```

Reply:
So when we remove the Types in the package.json, it then allows the compiler to build its own types instead of having them types be defined by a single file. 

So if we remove the type file and the line in the package.json. ergo it will make its own types for the other files to read!